app/assets/stylesheets/admin/dashboard.css.scss

```diff
@@ -218,10 +218,82 @@
 }
 
 .boundary-line {
+  border-bottom: 2px solid #aaa;
   padding-top: 30px;
+}
+
+.panel-boundary-bottom {
   border-bottom: 2px solid #aaa;
+  margin-left: 5px;
+  margin-right: 5px;
+  padding-top: 5px;
 }
 
 .book-title-table {
   max-width: 600px;
 }
+
+.panel-title {
+  font-size: 23px;
+  margin-left: 10px;
+  margin-top: 5px;
+  padding-bottom: 5px;
+}
+
+.book-search-panel-left {
+  border-right: 1px solid #ddd;
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.